app/database/operator/server_data_operator/handlers/category.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {MM_TABLES} from '@constants/database';
+import DataOperatorException from '@database/exceptions/data_operator_exception';
+import {
+    isRecordCategoryChannelEqualToRaw,
+    isRecordCategoryEqualToRaw,
+} from '@database/operator/server_data_operator/comparators';
+import {
+    transformCategoryChannelRecord,
+    transformCategoryRecord,
+} from '@database/operator/server_data_operator/transformers/category';
+import {getUniqueRawsBy} from '@database/operator/utils/general';
+
+import type {
+    HandleCategoryChannelArgs,
+    HandleCategoryArgs,
+} from '@typings/database/database';
+import type CategoryModel from '@typings/database/models/servers/category';
```

Comment:
```suggestion

import type CategoryModel from '@typings/database/models/servers/category';
```

Reply:
What is the reason for the newline here? ESLint doesn't give any errors as it is bunched up with other `import type`'s

Reply:
It should error as based on the sort imports there should be an empty line between regular imports and import types

Reply:
Ah it's also an import type;

```
import type {
    HandleCategoryChannelArgs,
    HandleCategoryArgs,
} from '@typings/database/database';
```